contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -129,7 +129,6 @@ contract BasicAMBErc677ToErc677 is
         external
```

Comment:
Could you imagine a scenario when it is required to use this functionality for this type of the bridge? Do we have a similar functionality for the opposite direction?

Reply:
Since it is a `BasicAMBErc677ToErc677.sol` contract, the functionality will be available on both sides of the bridge. 
I guess it makes sense to change a misleading parameter name `unlockOnForeign` to `unlockOnOtherSide`.